sources/packages/backend/libs/services/src/msfaa-number/msfaa-number-shared.service.ts

```diff
@@ -109,13 +109,16 @@ export class MSFAANumberSharedService {
    * @param offeringIntensity offering intensity.
    * @param auditUserId user that should be considered the one that is causing the changes.
    * individually based on, for instance, the Part time/Full time.
+   * @params options.
+   * - `msfaaNumber` reactivated msfaaNumber.
    * @returns created MSFAA record.
    */
-  private async internalCreateMSFAANumber(
+  public async internalCreateMSFAANumber(
```

Comment:
The method was called "internal" to enforce its `private` scope.
Maybe we can create a new "public" method to reactivate the MSFAA where the `msfaaNumberId` would not be optional. This method would then call the `internalCreateMSFAANumber`. We ca maybe rename the `internalCreateMSFAANumber` to `internalActivateMSFAANumber`.
By the way, we have been trying to avoid using the `public` keyword explicitly.